docs/source/preview_matplotlib_datasets.md

```diff
@@ -1,10 +1,10 @@
-# Visualise charts in Kedro-Viz with Matplotlib
+# Preview Matplotlib charts in Kedro-Viz 
 
-This page describes how to output interactive visualisations of a Kedro project with Kedro-Viz, which supports integration with [Matplotlib](https://matplotlib.org/). You can view Matplotlib charts in Kedro-Viz when you use the MatplotLibWriter dataset.
+This page describes how to output static visualisations of a Kedro project with Kedro-Viz, which supports integration with [Matplotlib](https://matplotlib.org/). You can view Matplotlib charts in Kedro-Viz when you use the MatplotLibWriter dataset.
```

Comment:
```suggestion
This page describes how to output static visualisations of a Kedro project with Kedro-Viz, which supports integration with [Matplotlib](https://matplotlib.org/). You can view Matplotlib charts in Kedro-Viz when you use the MatplotlibWriter dataset.
```

plus a link to https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-2.0.0/api/kedro_datasets.matplotlib.MatplotlibWriter.html